tests/TestInputSocket.py

```diff
@@ -137,14 +137,14 @@ def setUp(self):
     def sanityCheck(self, _tn):
         # Check that the connection has been established
         out = _tn.getOutput()
-        self.assertTrue(string.split(out, '\n')[0] == 'Connected to JSBSim server',
+        self.assertTrue(out.split('\n')[0] == 'Connected to JSBSim server',
                         msg="Not connected to the JSBSim server.\nGot message '%s' instead" % (out,))
 
         # Check that "help" returns the minimum set of commands that will be
         # tested
-        self.assertEqual(sorted(map(lambda x: string.strip(string.split(x, '{')[0]),
-                                    string.split(_tn.sendCommand("help"), '\n')[2:-2])),
-                         ['get', 'help', 'hold', 'info', 'iterate', 'quit', 'resume', 'set'])
+        self.assertEqual(sorted(map(lambda x: x.split('{').strip()[0]),
+                                    _tn.sendCommand("help").split('\n')[2:-2]),
+                                    ['get', 'help', 'hold', 'info', 'iterate', 'quit', 'resume', 'set'])
```

Comment:
According to my tests, this code has been incorrectly modified. It should rather be
```python
       self.assertEqual(sorted(map(lambda x: x.split('{')[0].strip(),
                                   _tn.sendCommand("help").split('\n')[2:-2])),
                               ['get', 'help', 'hold', 'info', 'iterate', 'quit', 'resume', 'set'])
```
Otherwise the test fails.